src/ledger/pathfind.ts

```diff
@@ -46,11 +47,12 @@ function requestPathFind(
     },
     pathfind.destination.amount
   )
-  const request: PathFindRequest = {
+  const request: RipplePathFindRequest = {
     command: 'ripple_path_find',
     source_account: pathfind.source.address,
     destination_account: pathfind.destination.address,
-    destination_amount: toRippledAmount(destinationAmount)
+    // @ts-ignore
+    destination_amount: destinationAmount
```

Comment:
Why are you ignoring the type here? (Curious)

Reply:
It's a mishmash of old types and new types, and it'll get overwritten when we rewrite the sugar anyways. This is a temporary measure.